common/app/routes/Challenges/Side-Panel.jsx

```diff
@@ -95,7 +97,8 @@ export class SidePanel extends PureComponent {
     this.descriptionTop = node;
   }
 
-  renderDescription(description = [ 'Happy Coding!' ]) {
+  renderDescription() {
+    const { description = [ 'Happy Coding!' ] } = this.props;
```

Comment:
I've seen you introducing these changes in other places (moving passed in args to props destructuring). They don't seem to be related to the PR they are in. Is there something I'm missing?

This is a pattern I intentionally use where possible so that you only need to look at the render method to see where props are being used.